gsoc/2021/posts/index.md

```diff
@@ -66,6 +66,7 @@ Mentors: Kanav Gupta, Aditya Aggarwal
 1. [Introduction](/web/gsoc/2021/posts/trung_ngo_tan/post01)
 2. [Fast inference](/web/gsoc/2021/posts/trung_ngo_tan/post02)
 3. [1st Phase report](/web/gsoc/2021/posts/trung_ngo_tan/post03)
+4. [Body and Hand detection + Image base recognition (Installation and Usage)] (/web/gsoc/2021/posts/trung_ngo_tan/post04)
```

Comment:
The link will not work with the space here. Can you please update that?

Reply:
I have just fixed it, please check in the newest pull request.